src/main/java/io/orangebeard/listener/OrangebeardTestSystemListener.java

```diff
@@ -1,141 +1,159 @@
-import com.epam.ta.reportportal.ws.model.FinishTestItemRQ;
-import com.epam.ta.reportportal.ws.model.StartTestItemRQ;
-import com.epam.ta.reportportal.ws.model.attribute.ItemAttributesRQ;
-import com.epam.ta.reportportal.ws.model.launch.Mode;
-import com.epam.ta.reportportal.ws.model.launch.StartLaunchRQ;
-import com.google.api.client.http.*;
-import com.google.api.client.http.javanet.NetHttpTransport;
 import fitnesse.testrunner.WikiTestPage;
-import fitnesse.testsystems.*;
+import fitnesse.testsystems.Assertion;
+import fitnesse.testsystems.ExceptionResult;
+import fitnesse.testsystems.TestPage;
+import fitnesse.testsystems.TestResult;
+import fitnesse.testsystems.TestSummary;
+import fitnesse.testsystems.TestSystem;
+import fitnesse.testsystems.TestSystemListener;
 import fitnesse.wiki.PageData;
 import fitnesse.wiki.WikiPageProperty;
+import io.orangebeard.client.OrangebeardClient;
+import io.orangebeard.client.OrangebeardProperties;
+import io.orangebeard.client.entity.Attribute;
+import io.orangebeard.client.entity.FinishTestItem;
+import io.orangebeard.client.entity.FinishTestRun;
+import io.orangebeard.client.entity.StartTestItem;
+import io.orangebeard.client.entity.StartTestRun;
+import io.orangebeard.client.entity.Status;
+import io.orangebeard.client.entity.TestItemType;
 import io.orangebeard.listener.helper.OrangebeardLogger;
 import io.orangebeard.listener.helper.OrangebeardTableLogParser;
 import io.orangebeard.listener.helper.TestPageHelper;
 import io.orangebeard.listener.helper.ToolchainRunningContext;
-import io.reactivex.Maybe;
 import lombok.NoArgsConstructor;
-import org.slf4j.LoggerFactory;
+import lombok.SneakyThrows;
 
+import javax.annotation.Nullable;
 import java.io.Closeable;
+import java.io.File;
 import java.io.IOException;
-import java.time.Instant;
-import java.util.*;
+import java.net.InetAddress;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+import java.util.UUID;
 import java.util.stream.Collectors;
 
-import static java.lang.String.format;
+import static io.orangebeard.listener.helper.TestPageHelper.getTestName;
 import static java.util.Objects.requireNonNull;
 
 @NoArgsConstructor
 public class OrangebeardTestSystemListener implements TestSystemListener, Closeable {
+    private String PROPERTY_FILE_NAME = "orangebeard.properties";
+    private static final String PROP_ROOT_PATH = "fitnesseroot.path";
 
-    private static final String PROP_LAUNCHNAME = "rp.launch";
-    private static final String PROP_AUGMENTED_ENDPOINT = "at.augmented-testing.endpoint";
-    private static final String PROP_AUGMENTED_TOKEN = "at.augmented-testing.token";
-    private static final String AUGMENTED_TESTING_TOKEN = "AUGMENTED_TESTING_TOKEN";
+    private OrangebeardClient orangebeard;
+    private final OrangebeardProperties orangebeardProperties = new OrangebeardProperties();
```

Comment:
Wat doen we eigenlijk nog met die property file name als je heir de properties zonder dat ding instantieert?

Reply:
in exotische projectinrichtingen kun je voor de listener het fitnesse root path meegeven, dat is nog de enige reden om ook hier de propertyfile te kunnen definieren